components/02-elements/badge/badge.hbs

```diff
@@ -1,3 +1,7 @@
-<div class="badge {{ class }}">
+{{#if modifier}}
+    <div class="badge badge--{{ modifier }}">
+{{else}}
+    <div class="badge">
+{{/if}}
     {{ text }}
 </div>
```

Comment:
I'd use the old version of code here, with classes, b/c 1. you don't have to use `if/else` statement, 2. we use this 'class' approach through whole alpaca components project.

Reply:
Done.